src/Fixer/Naming/HomoglyphNamesFixer.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Naming;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Fred Cox <mcfedr@gmail.com>
+ */
+final class HomoglyphNamesFixer extends AbstractFixer
+{
+    /**
+     * Used the site https://github.com/mcfedr/homoglyph-download.
```

Comment:
1/2/3/4. I came to this because of issues with cyrillic letters in my company, I went though a number of datasets, and looked at what the options are. At first I though I'd use the set from https://github.com/codebox/homoglyph - I forked it to add a php exporter.

But since then I came across the set at http://homoglyphs.net and decided it was more appropriate - its much smaller, and I felt focused on letters that a) look identical, not just similar b) are more likely to have been used accidentally - the full set contains symbols such as `①` which you are unlikely to type on your keyboard and unlikely to confuse with `1`

5. I put it in a seperate `Helper` class at first, but not sure where best in the project to put it. I think it makes sense for it to be PHP.